scripts/build-testing-sdk.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * This script bundles some scripts used for testing:
+ * - amd loader
+ * - polyfills
+ *   + fetch
+ *   + promises
+ *   + bace64 encoding
+ * - then deps for the testing framework
+ */
+const fs = require('fs');
+
+const sourceModules = getSourceFromModules([
+  // umd module loader
+  'loader.js',
+  // original testing sdk deps
+  'route-recognizer',
+  'fake-xml-http-request',
+  'pretender'
+]);
+
+const outSource = `
+${sourceModules[0]}
+(function (self) {
+;self.fetch = null;
+${sourceModules.slice(1).join('\n')}
+}(window));
+`;
+
+console.log(outSource);
+
+function getSourceFromModules(modules) {
+
+  return modules
```

Comment:
@minasmart I may have been chaining calls beyond map. Generally if I chain I've always dropped to a newline even for the first call to indicate I'll be chaining. In this case it's a mistake. I will fix... I've also noticed even when chaining people here don't drop to a newline so that'll have to be a muscle memory fix thing for me to figure out.